### ISSUE TITLE
No support for disable referential integrity

### ISSUE THREAD

**Issue by zjuanma** [ibmdb/ruby-ibmdb#39]
To run minitest with fixtures is necessary to support disable referential integrity.

**Comment by zjuanma** [comment]
Patch:

```
module ActiveRecord
  module ConnectionAdapters
    class IBM_DBAdapter
      def supports_disable_referential_integrity? #:nodoc:
          true
      end

      def disable_referential_integrity #:nodoc:
        if supports_disable_referential_integrity?
          alter_foreign_keys(tables, true)
        end
        yield
      ensure
        if supports_disable_referential_integrity?
          alter_foreign_keys(tables, false)
        end
      end

      def alter_foreign_keys(tables, not_enforced)
        enforced = not_enforced ? 'NOT ENFORCED' : 'ENFORCED'
        tables.each do |table|
          foreign_keys(table).each do |fk|
            execute("ALTER TABLE #{fk.from_table.upcase} ALTER FOREIGN KEY #{fk.name.upcase} #{enforced}")
          end
        end
      end
    end
  end
end
```

**Comment by arvindgu** [comment]
@zjuanma  Thank you for the patch, we have released ibm_db gem version 3.0.0 and added above patch.
